packages/playwright/src/reporter.ts

```diff
@@ -90,7 +94,7 @@ class ReplayPlaywrightReporter implements Reporter {
     this.config = config;
     this.reporter = new ReplayReporter(
       {
-        name: "playwright",
```

Comment:
Shouldn't the package name and version be imported from the `package.json` ? That way they'll stay in sync.

That's what we're doing in the other packages:

https://github.com/replayio/replay-cli/blob/db3f422416d9385db543e9d4876752a159adfa79/packages/replayio/src/bin.ts#L4

https://github.com/replayio/replay-cli/blob/db3f422416d9385db543e9d4876752a159adfa79/packages/replayio/src/bin.ts#L20